epochStart/metachain/systemSCs.go

```diff
@@ -417,7 +417,17 @@ func (s *systemSCProcessor) fillStakingDataForNonEligible(validatorInfos map[uin
 
 			err := s.stakingDataProvider.FillValidatorInfo(validatorInfo.PublicKey)
 			if err != nil {
-				return err
+				log.Error("fillStakingDataForNonEligible", "error", err)
+				if validatorInfo.List == string(core.WaitingList) || validatorInfo.List == string(core.JailedList) {
+					return err
+				}
+
```

Comment:
I would add a log.Error here as this should not happen and we should correct this.

Reply:
log . error is on line 420

Reply:
log.error is on line 420